NitroxClient/GameLogic/PlayerModelBuilder/PlayerModelDirector.cs

```diff
@@ -0,0 +1,35 @@
+﻿using NitroxClient.GameLogic.PlayerModelBuilder.Abstract;
+using UnityEngine;
+
+namespace NitroxClient.GameLogic.PlayerModelBuilder
+{
+    // The vision here is to use this director as our central tool for managing remote model appearance alterations like changing equipment or in-game changes to appearance.
+    // This could probably very easily be extended to work with the local Player later down the line.
+    public class PlayerModelDirector
```

Comment:
Is this extra level of indirection necessary? I don't see any added value in `StagePlayer()` - this could be put in the constructor, and then (similar to Android semantics):

```cs
new PlayerModel.Builder(player)
    .WithPing()
    .WithRegularDiveSuit()
    .Build();
```
Or even more advanced:
```cs
RemotePlayer player = new RemotePlayer.Builder(playerId, playerName, playerSettings)
    .WithPing()
    .WithRegularDiveSuit()
    .Build();
```

Reply:
The Director is an intrinsic artifact of the Builder pattern.